include/deal.II/physics/elasticity/standard_tensors.h

```diff
@@ -132,12 +132,12 @@ namespace Physics
        *  \mathcal{P} : \{ \bullet \}
        *  := \{ \bullet \} - \frac{1}{\textrm{dim}} \left[ \{ \bullet \} : \mathbf{I} \right]\mathbf{I}
        *   = \mathcal{P}^{T} : \{ \bullet \}
-       *   = \texttt{dev_P} \left( \{ \bullet \} \right)
+       *   = \texttt{dev\_P} \left( \{ \bullet \} \right)
```

Comment:
You would never have guessed that I read over this about 100 times before it was merged...